graphql-cli/src/test/java/io/ballerina/graphql/cmd/SdlSchemaGenerationTest.java

```diff
@@ -144,19 +134,16 @@ public void testSdlGenerationWithMultipleServices2() {
             expectedSchema = readContentWithFormat(expectedSchemaFile);
             generatedSchema = readContentWithFormat(this.tmpDir.resolve("schema_service4_2.graphql"));
             Assert.assertEquals(expectedSchema, generatedSchema);
-        } catch (BLauncherException | IOException e) {
+        } catch (IOException | InterruptedException e) {
             Assert.fail(e.toString());
         }
     }
 
     @Test(description = "Test successful GraphQL command execution with multiple services in a bal project")
     public void testSdlGenerationWithMultipleServicesInProject() {
-        Path graphqlService = resourceDir.resolve(Paths.get("graphqlServices/valid/project_2", "main.bal"));
-        String[] args = {"-i", graphqlService.toString(), "-o", this.tmpDir.toString()};
-        GraphqlCmd graphqlCmd = new GraphqlCmd(printStream, tmpDir, false);
-        new CommandLine(graphqlCmd).parseArgs(args);
+        String[] args = {"-i", "valid/project_2/main.bal", "-o", this.tmpDir.toString()};
         try {
-            graphqlCmd.execute();
+            executeCommand(args);
             Assert.assertTrue(Files.exists(this.tmpDir.resolve("schema_main.graphql")));
             Path expectedSchemaFile = resourceDir.resolve(Paths.get("expectedSchemas", "schema_service4.graphql"));
```

Comment:
@DimuthuMadushan Shall we rename these `.graphql` files with additional underscore. ex: `schema_service_4.graphql`

Reply:
Fixed in https://github.com/ballerina-platform/graphql-tools/pull/136/commits/20e28198e8bb76762c77a9ef29879c24a2af0ab1